### PR TITLE
Updates to UG

### PR DESCRIPTION
**Pull request by Preshita01**
I also went through the whole UG and edited some parts of it to make it more user friendly.

### CODE REVIEW

File: docs/UserGuide.md
Comment (JunWei3112) [inline docs/UserGuide.md:358]: Just a minor nitpick here! Can considering changing from `tutors` to `you` to make it more personalised

File: docs/UserGuide.md
Comment (JunWei3112) [inline docs/UserGuide.md:365]: Can consider removing this sentence since it is already mentioned

File: docs/UserGuide.md
Comment (JunWei3112) [inline docs/UserGuide.md:381]: A very very minor nitpick here, but can consider adding a period `.` at the end at the sentence for consistency!

**Review by JunWei3112 — COMMENTED** [review]
Just some minor issues to be fixed. Else, LGTM!

File: docs/UserGuide.md
Comment (wpinrui) [inline docs/UserGuide.md:66]: The 23 wasn't a typo - the hotlink omits the dot. Can you confirm this link still works?

**Review by wpinrui — APPROVED** [review]
LGTM after merge conflicts are fixed

File: docs/UserGuide.md
Comment (Preshita01) [reply docs/UserGuide.md]: Right, I read up a bit on it and realised it is better to leave it as `23` as well, thank you for highlighting!

**Review by JunWei3112 — APPROVED** [review]
LGTM!

**Pull request merged**
